.github/workflows/ci.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     steps:
     # Checks-out the repo. More at: https://github.com/actions/checkout
     - uses: actions/checkout@v2
+    - name: Select Xcode version
```

Comment:
Maybe we can comment that this can be removed eventually?

Reply:
I think we want to keep being explicit about what Xcode version we're using given the Swift requirement. I hate that we need to do this so hacky, but I didn't find a better way to do it with GitHub actions.